src/main/java/com/rednavis/vaadin/showcase/backend/service/password/PasswordServiceImpl.java

```diff
@@ -0,0 +1,135 @@
+package com.rednavis.vaadin.showcase.backend.service.password;
+
+import com.rednavis.vaadin.showcase.backend.config.PasswordConfig;
+import edu.vt.middleware.password.AlphabeticalCharacterRule;
+import edu.vt.middleware.password.AlphabeticalSequenceRule;
+import edu.vt.middleware.password.DigitCharacterRule;
+import edu.vt.middleware.password.LengthRule;
+import edu.vt.middleware.password.LowercaseCharacterRule;
+import edu.vt.middleware.password.NonAlphanumericCharacterRule;
+import edu.vt.middleware.password.NumericalSequenceRule;
+import edu.vt.middleware.password.Password;
+import edu.vt.middleware.password.PasswordData;
+import edu.vt.middleware.password.PasswordValidator;
+import edu.vt.middleware.password.QwertySequenceRule;
+import edu.vt.middleware.password.RepeatCharacterRegexRule;
+import edu.vt.middleware.password.Rule;
+import edu.vt.middleware.password.RuleResult;
+import edu.vt.middleware.password.UppercaseCharacterRule;
+import edu.vt.middleware.password.WhitespaceRule;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Singleton
+public class PasswordServiceImpl implements PasswordService {
+
+  private final PasswordValidator passwordValidator;
+
+  @Inject
+  public PasswordServiceImpl(PasswordConfig passwordConfig) {
```

Comment:
Please consult with Dmitry Tsydzik on how to improve that and what pattern could help. Probably we could use Lambda and Predicate to execute some Command if Predicate is positive.